web-components/src/components/inputs/CheckboxLabel.tsx

```diff
@@ -47,7 +49,11 @@ export default function CheckboxLabel({
         label={isLabelStringOrNumber ? <Subtitle>{label}</Subtitle> : label}
       />
       {showError && (
-        <FormHelperText error={showError}>{fieldError}</FormHelperText>
+        <FormHelperText error={showError} sx={{ ml: 9 }}>
+          <Subtitle size="sm" sx={{ color: theme.palette.error.main }}>
+            {fieldError}
+          </Subtitle>
+        </FormHelperText>
```

Comment:
check the console for errors here - might give you a `no nesting` error or warning. if that happens, you can set the subtitlte to render `as=“span”`

Reply:
Nice! thanks @haveanicedavid